ocs_ci/helpers/helpers.py

```diff
@@ -56,7 +57,8 @@ def create_unique_resource_name(resource_description, resource_type):
     Returns:
         str: A unique name
     """
-    return f"{resource_type}-{resource_description[:23]}-{uuid4().hex}"
+    name = f"{resource_type}-{resource_description[:23]}-{uuid4().hex}"
+    return name if len(name) < 64 else name[:63]
```

Comment:
I think it's too risky to leave the length so close to the maximum possible.
IMO, 40 characters should be sufficient, and it'll also help keep naming more clean and digestible